FMRApp/protocol/lv_FMR.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--Initially generated by $Id$ -->
+<lvinput xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://epics.isis.rl.ac.uk/lvDCOMinput/1.0" xsi:schemaLocation="http://epics.isis.rl.ac.uk/lvDCOMinput/1.0 lvDCOMinput.xsd">
+  <!--Full path to external interface vi used for triggering LabVIEW events when extint="true" below -->
+  <extint path="$(LVDCOM=$(TOP))/lvDCOMApp/src/extint/Main/Library/External Interface - Set Value.vi">
+    <!--Look for extint in LVDCOM directory or TOP if not defined-->
+  </extint>
+  <!--Definition of a configSection, the second argument of the IOC lvDCOMConfigure() command -->
+  <section name="frontpanel">
+    <!--TODO: edit path attribute in <vi /> to contain a full absolute path, you can also use any EPICS macro known to the IOC e.g. $(TOP) or $(LVDCOM)-->
+    <vi path="C:/Users/wil0322903/Desktop/VNA-FMR/VNA-FMR-POLREF.vi">
```

Comment:
This path isn't right. Can you point it at the VI within this folder? Or if it's going to change make it a macro.

Reply:
Macro added.